addon/components/ui-button/component.ts

```diff
@@ -26,6 +27,12 @@ export default class UiButton extends Component {
   @localClassName()
   isLoading?: boolean = false;
 
+  @attribute()
+  @computed("width")
+  get style(): string {
+    return `width:${this.width}`;
+  }
+
```

Comment:
Does this not give errors/warning? 

Don't you have to escape this output before inserting it into the template (ember safestring, etc)?  What are we doing in other components in EA?